src/webextension/widgets/input.css

```diff
@@ -104,3 +108,44 @@
   margin: 10px 0 3px 0;
   font-size: 13px;
 }
+
+.filter {
+  border: 1px solid #b6b6b6;
```

Comment:
I don't think this should be necessary; the difference between this and the border color of all the other inputs (`rgba(12, 12, 13, 0.5)`) is hard to tell, and in any case, I think we should aim for consistency in our widgets.